mscore/startcenter.h

```diff
@@ -18,7 +18,7 @@
 
 namespace Ms {
 
-#if 0
+#if USE_WEBENGINE
```

Comment:
But then again see http://stackoverflow.com/questions/135069/ifdef-vs-if-which-is-better-safer-as-a-method-for-enabling-disabling-compila, several good reasons to be using `#if`